google/resource_storage_default_object_access_control_test.go

```diff
@@ -0,0 +1,110 @@
+package google
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccStorageDefaultObjectAccessControl_basic(t *testing.T) {
```

Comment:
Is this basic test duplicated by the generated test?

Reply:
Yep! I'm gonna fry all the duplicated tests in the future, I just haven't taken the time to do it quite yet.